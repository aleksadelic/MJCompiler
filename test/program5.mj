program MJProgram
	int x; int y;
	int arr[];
	int arrNew[];
	int i, j;
{

	void main() {
		
		arr = new int[5];
		
		i = 0;
		while (i < 5) {
			arr[i] = i + 1;
			i++;
		}
		
		i = 0;
		while (i < 5) {
			print(arr[i]);
			i++;
		} 
			
		arrNew = arr.map(x => 2 * x + 10 * x);
		
		i = 0;
		while (i < 5) {
			print(arrNew[i]);
			i++;
		}
		
	}
	
}