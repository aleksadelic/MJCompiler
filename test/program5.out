    1    2    3    4    5   12   24   36   48   60
Completion took 7 ms